README.md

```diff
@@ -428,3 +428,7 @@ Example 4: After P-1♣️!-1♥️-1♠️-3♣️!, note that the initial pass
           deny the asked queen but show another queen.
     * DOPI over interference (Double = 0 or 3, Pass = 1 or 4, Step = 2 or 5 w/o
       queen, 2 steps = 2 or 5 w/ queen, assuming space permits)
+* Gerber
+    * Jump from 1NT or 2NT openings or pseudo-openings to 4♣️ for Ace ask. Responses use the same step increments as Blackwood. 5♣️ is King ask with the same Blackwood-esque stepped responses.
```

Comment:
1) Please wrap text like in the rest of the file.
2) "same step increments as Blackwood" is confusing because RKC is a variant of blackwood, so it's not clear if you mean 4D = 0 or 4 or if the steps are 1430.
3) Specific king responses are probably better than king quantity, since king asks are usually only used for investigating grands.